src/main/webapp/js/instructorCoursesAjax.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function(){
+    var ajaxRequest = function(e) {
+        e.preventDefault();
+        var formData = $(this).serialize();
+        $.ajax({
+            type: 'POST',
+            cache: false,
+            url: $(this).attr('action') + '?' + formData,
+            beforeSend: function() {
+                $('#coursesList').html(
+                    '<img height="75" width="75" class="margin-center-horizontal" src="/images/ajax-preload.gif"/>'
+                );
+            },
+            error: function() {
+
```

Comment:
Do you mean by showing a button for user to retry loading the courses list if the request fails ? Or we can just show a message telling user to reload the page ? This one might need to be modified for the `Sessions` page also since it currently does not handle that